WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -134,12 +221,28 @@ class SiteDesignContentCollectionViewController: CollapsableHeaderViewController
     }
 
     private enum TextContent {
-        static let mainTitle = NSLocalizedString("Choose a theme", comment: "Title for the screen to pick a theme and homepage for a site.")
-        static let backButtonTitle = NSLocalizedString("Design", comment: "Shortened version of the main title to be used in back navigation.")
-        static let skipButtonTitle = NSLocalizedString("Skip", comment: "Continue without making a selection.")
-        static let cancelButtonTitle = NSLocalizedString("Cancel", comment: "Cancel site creation.")
-        static let errorTitle = NSLocalizedString("Unable to load this content right now.", comment: "Informing the user that a network request failed becuase the device wasn't able to establish a network connection.")
-        static let errorSubtitle = NSLocalizedString("Check your network connection and try again.", comment: "Default subtitle for no-results when there is no connection.")
+        static let mainTitle = NSLocalizedString("Choose a theme",
+                                                 comment: "Title for the screen to pick a theme and homepage for a site.")
+        static let backButtonTitle = NSLocalizedString("Design",
+                                                       comment: "Shortened version of the main title to be used in back navigation.")
+        static let skipButtonTitle = NSLocalizedString("Skip",
+                                                       comment: "Continue without making a selection.")
+        static let cancelButtonTitle = NSLocalizedString("Cancel",
+                                                         comment: "Cancel site creation.")
+        static let errorTitle = NSLocalizedString("Unable to load this content right now.",
+                                                  comment: "Informing the user that a network request failed becuase the device wasn't able to establish a network connection.")
+        static let errorSubtitle = NSLocalizedString("Check your network connection and try again.",
+                                                     comment: "Default subtitle for no-results when there is no connection.")
+        static let helperText = NSLocalizedString("Can’t decide? You can change the theme at any time",
```

Comment:
On the related Android PR, the [text ends with a period](https://github.com/wordpress-mobile/WordPress-Android/pull/16555/files#diff-6bbab92723bb83a1300e86d496d62b9475a6a1e8b5a5feca27721edfbb09943fR3399), since it seemed to match better the style used elsewhere (e.g. [the intent header subtitle](https://github.com/wordpress-mobile/WordPress-Android/blob/381266508e5f0e1b655eeb74b7bdcc4a48e38682/WordPress/src/main/res/values/strings.xml#L3264)). Otoh, this text without the period matches the text in the design. Nitpick, either way, so I don' think this should be blocking.

Reply:
Good eye @mkevins! Pinging @kyleaparker to get confirmation on whether we should stick to the designs on this or add a period.

Reply:
Let's add the period 👍